arcgis-ios-sdk-samples/Edit data/Create and save KML file/CreateAndSaveKMLViewController.swift

```diff
@@ -0,0 +1,271 @@
+// Copyright © 2020 Esri.
```

Comment:
Not a big deal - should there be a trailing period on this line? I saw different versions from different samples. If yes, some samples below need to add it.